plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -188,8 +188,7 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 			throw new CoreException(new Status(IStatus.ERROR, CDKCoreActivator.PLUGIN_ID, "Call to minishift up has failed."));
 		}
 
-		IProcess process = addProcessToLaunch(p, launch,s);
-		linkTerminal(p);
```

Comment:
There are 2 subclasses to this file...  the 2.x will still use terminal, not console.  The minishift / cdk3 will use console. So this abstract superclass must still support terminal linkage.